example/storage/eventstore_test.go

```diff
@@ -40,9 +42,15 @@ const (
 
 func TestPostcard_Repositories(t *testing.T) {
 	conn := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable", host, port, user, password, dbname)
-	db, err := sql.Open("postgres", conn)
+	postgresDB, err := sql.Open("postgres", conn)
 	require.NoError(t, err)
 
+	dbFile, err := os.CreateTemp("", "tmpfile-")
+	if err != nil {
+		log.Fatal(err)
+	}
+	sqliteDB, err := sql.Open("sqlite3", dbFile.Name())
```

Comment:
is there no lib to `import _ sqlite` for the driver?

Reply:
There is. The final version uses that.